package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vanilla-framework",
-  "version": "3.12.0",
+  "version": "3.12.1",
```

Comment:
We haven't released 3.12.0, so you don't need to do it. Sorry, I know it's not exactly clear when to do it.